noxfiles/ci_nox.py

```diff
@@ -162,182 +166,156 @@ def minimal_config_startup(session: nox.Session) -> None:
-        nox.param("saas", id="saas"),
-    ],
-)
-def pytest_ops(session: nox.Session, mark: str) -> None:
-    """Runs fidesops tests."""
-    session.notify("teardown")
-    if mark == "unit":
-        session.run(*START_APP, external=True)
-        run_command = (
-            *RUN_NO_DEPS,
-            "pytest",
-            OPS_TEST_DIR,
-            "-m",
-            "not integration and not integration_external and not integration_saas",
-        )
-        session.run(*run_command, external=True)
-    elif mark == "integration":
-        run_infrastructure(
-            run_tests=True,
-            analytics_opt_out=True,
-            datastores=[],
-            pytest_path=OPS_TEST_DIR,
-        )
-    elif mark == "external_datastores":
-        session.run(*START_APP, external=True)
-        run_command = (
-            "docker",
-            "compose",
-            "run",
-            "-e",
-            "ANALYTICS_OPT_OUT",
-            "-e",
-            "REDSHIFT_TEST_URI",
-            "-e",
-            "SNOWFLAKE_TEST_URI",
-            "-e",
-            "REDSHIFT_TEST_DB_SCHEMA",
-            "-e",
-            "BIGQUERY_KEYFILE_CREDS",
-            "-e",
-            "BIGQUERY_DATASET",
-            "--rm",
-            CI_ARGS,
-            COMPOSE_SERVICE_NAME,
-            "pytest",
-            OPS_TEST_DIR,
-            "-m",
-            "integration_external",
-        )
-        session.run(*run_command, external=True)
-    elif mark == "saas":
-        # This test runs an additional integration Postgres database.
-        # Some connectors cannot be traversed with the standard email
-        # identity and require another dataset to provide a starting value.
-        #
-        #         ┌────────┐                 ┌────────┐
-        # email──►│postgres├──►delivery_id──►│doordash│
-        #         └────────┘                 └────────┘
-        #
-        session.run(*START_APP_WITH_EXTERNAL_POSTGRES, external=True)
-        run_command = (
-            "docker",
-            "compose",
-            "run",
-            "-e",
-            "ANALYTICS_OPT_OUT",
-            "-e",
-            "VAULT_ADDR",
-            "-e",
-            "VAULT_NAMESPACE",
-            "-e",
-            "VAULT_TOKEN",
-            "--rm",
-            CI_ARGS,
-            COMPOSE_SERVICE_NAME,
-            "pytest",
-            OPS_TEST_DIR,
-            "-m",
-            "integration_saas",
-        )
-        session.run(*run_command, external=True)
+def sum_coverage(session: nox.Session) -> None:
+    """
+    Combine multiple Code Climate coverage files.
+
+    7 files are expected for summation, one from each of the test groups run in CI.
+    """
+    download_cc_reporter = f"curl -L {TEST_REPORTER_URL} > ./cc-test-reporter && chmod +x ./cc-test-reporter && "
+    sum_test_coverage = (
+        "./cc-test-reporter sum-coverage --parts 4 coverage/**/codeclimate.json ;"
```

Comment:
The comment says 7 files, but `--parts` is set to 4. Should this be 7?

Reply:
the comment is wrong, I updated it to _not_ expect the external tests and forgot to update the comment

Reply:
fixed! good catch